modules/grafana/manifests/dashboards.pp

```diff
@@ -39,6 +39,7 @@
     "${dashboard_directory}/edge_health.json": content => template('grafana/dashboards/edge_health.json.erb');
     "${dashboard_directory}/origin_health.json": content => template('grafana/dashboards/origin_health.json.erb');
     "${dashboard_directory}/whitehall_health.json": content => template('grafana/dashboards/whitehall_health.json.erb');
+    "${dashboard_directory}/whitehall_5xx_analysis.json": content => template('grafana/dashboards/whitehall_5xx_analysis.json.erb');
```

Comment:
This doesn't look to use any of the templating functions, so you might want to just add it as a file.

However, some of the other dashboards that use Elasticsearch do use templating, so it might be good to look at why that is used.

Reply:
Seems the templating used by the other ERB dashboards is to use `fields_prefix` but we know that whitehall doesn't use one so I reckon we're ok without doing that here.  Most of the other ERB dashboards don't actually use templating either, do you know if we're planning to move them all to files rather than templates?

I could argue for leaving them as templates, so it's less noisy if we _do_ need to introduce templating features, but it's not a hill I want to die on.

Reply:
> Seems the templating used by the other ERB dashboards is to use fields_prefix but we know that whitehall doesn't use one so I reckon we're ok without doing that here.

Sounds good.

> Most of the other ERB dashboards don't actually use templating either, do you know if we're planning to move them all to files rather than templates?

I put up a Pull Request for that change a couple of days ago, just because it seemed a bit neater [1].

1: https://github.com/alphagov/govuk-puppet/pull/7349

> I could argue for leaving them as templates, so it's less noisy if we do need to introduce templating features, but it's not a hill I want to die on.

I don't mind particularly either, just though I'd mention it.

Reply:
I hadn't spotted that PR, if this is the direction of travel, I'll follow suit.

Reply:
done in a rebase.

Reply:
Ok :+1: